plugin/core/host.py

```
@@ -0,0 +1,2032 @@
+    logger.debug("[Plugin Process] Sending response for req_id={}, success={}", req_id, ret_payload.get("success"))
+    try:
+        res_queue.put(ret_payload, timeout=10.0)
+        logger.debug("[Plugin Process] Response sent successfully for req_id={}", req_id)
+    except Exception:
+        logger.exception("[Plugin Process] Failed to send response for req_id={}", req_id)
+
+
+def _handle_config_update_command(
+    msg: dict,
+    ctx: Any,
+    instance: Any,
```

Ruff (0.15.1)

[warning] Unused function argument: `instance`

(ARG001)